### PR TITLE
Mailchimp Mailer Bridge: undefined index: _id when sending an email

### PR DESCRIPTION
**Pull request by monteiro**
| Q             | A
| ------------- | ---
| Branch?       | master for features / 4.4 or 5.0 for bug fixes 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | (no ticket created)
| License       | MIT

When I set the DSN:
`MAILER_DSN=mandrill://KEY@default`

and try to send an email:

```php
$reportEmail = (new Email())
            ->subject('this is a test')
            ->addFrom('test@test.com')
            ->addTo('test-2@test.com')
            ->text('this is just a test.');

$mailer->send($reportEmail);        
```

it returns an exception:

Captura de ecrã 2019-11-25, às 14 31 59

Stacktrace:

```
ErrorException:
Notice: Undefined index: _id

  at /Users/mac/Documents/workspace/spotahome/code/opensource/symfony/src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php:64
  at Symfony\Component\Mailer\Bridge\Mailchimp\Transport\MandrillHttpTransport->doSendHttp(object(SentMessage))
     (/Users/mac/Documents/workspace/spotahome/code/opensource/symfony/src/Symfony/Component/Mailer/Transport/AbstractHttpTransport.php:71)
  at Symfony\Component\Mailer\Transport\AbstractHttpTransport->doSend(object(SentMessage))
     (/Users/mac/Documents/workspace/spotahome/code/opensource/symfony/src/Symfony/Component/Mailer/Transport/AbstractTransport.php:71)
  at Symfony\Component\Mailer\Transport\AbstractTransport->send(object(SentMessage), object(DelayedEnvelope))
     (/Users/mac/Documents/workspace/spotahome/code/opensource/symfony/src/Symfony/Component/Mailer/Transport/Transports.php:51)
  at Symfony\Component\Mailer\Transport\Transports->send(object(Email), null)
     (/Users/mac/Documents/workspace/spotahome/code/opensource/symfony/src/Symfony/Component/Mailer/Mailer.php:41)
  at Symfony\Component\Mailer\Mailer->send(object(Email))
     (src/Controller/MailerController.php:20)
  at App\Controller\MailerController->index(object(Mailer))
```

## The fix, how does it work:

According to [documentation](https://mandrillapp.com/api/docs/messages.html), the endpoint `https://mandrillapp.com/api/1.0/messages/send-raw.json` returns an array of results in case of success:

```
[
    {
        "email": "recipient.email@example.com",
        "status": "sent",
        "reject_reason": "hard-bounce",
        "_id": "abc123abc123abc123abc123abc123"
    }
]
``` 

in case of an error:

```
{
    "status": "error",
    "code": 12,
    "name": "Unknown_Subaccount",
    "message": "No subaccount exists with the id 'customer-123'"
}
```

The fix just sets the right message id.

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php [hunk 1]
```diff
@@ -52,6 +52,7 @@ protected function doSendHttp(SentMessage $message): ResponseInterface
         ]);
 
         $result = $response->toArray(false);
+
```
Comment (fabpot) [inline src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php]: Can you revert this change?

**Review by fabpot — CHANGES_REQUESTED** [review]
Looks good to me. Can you change the target branch to 4.4 instead?

File: src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php (hunk 1 above)
Comment (monteiro) [reply src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php]: Absolutely!

**Comment by monteiro** [comment]
let me just create another PR.

**Comment by monteiro** [comment]
@fabpot just created https://github.com/symfony/symfony/pull/34599 👍
